content/blog/2023/2023_guide_to_speaking.md

```diff
@@ -0,0 +1,55 @@
+Title: Guide to Speaking at PyCon India
+Category: 2023
+Slug: guide-to-speaking
+
+**Hello Hello Pythonistas!!!**
+
+*Are you all ready to witness the largest gathering of Pythonistas in India?*
+
+The largest gathering of Pythonistas in India, [**PyCon India**](https://in.pycon.org/2023/) is back. The 14th PyCon India will be held online 29 Sept - 02 Oct in Hyderabad. So, whether you're someone who is submitting a presentation idea for the first time or you want to share what you have been working on with Python for a while or just want to share your tales with Python, this is the place for you. Your moment to shine has come!
+
+We're excited to see beginning, intermediate, and advanced proposals on a range of topics from speakers with varying experience and skillsets. You don't have to be a veteran speaker to submit a proposal. We are seeking people of all backgrounds to help our conference be really diverse. 
+
+**Why should you speak at PyCon India?**
+
+1. Get a chance to contribute to the Python community
+2. Share your Experiences and Perspectives with a Diverse Audience
+3. Receive valuable feedback & network with Python Experts
+4. Foster Innovation & Inspire others!
+
+Now that we have understood why one should submit a CFP, lets quickly dive into the things to keep in mind when drafting your CFP. We will discuss how to submit your CFP and provide some steps for new speakers to take the stage at PyCon India.
+
+- ***Keep a track of the Deadlines***: Keep an eye on the deadline so that you don't forget on missing to submit the proposal. Start preparing now and avoid the last minute hassel.
```

Comment:
```suggestion
- ***Keep track of the Deadlines***: Keep an eye on the deadline so that you don't forget on missing to submit the proposal. Start preparing now and avoid the last-minute hassle.
```